src/components/Welcome.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useLocation } from 'react-router-dom'
+
+const Welcome = () => {
+    const location = useLocation()
+    const username = location.state.username
```

Comment:
For the moment is ok to use it like that, but in the future we will implement Redux for access to a global state